acorn/src/unicode-property-data.ts

```diff
@@ -35,24 +35,18 @@ const unicodeScriptValues = {
   13: ecma13ScriptValues
 }
 
-const data = {}
-function buildUnicodeData(ecmaVersion) {
-  const d = data[ecmaVersion] = {
+export default Object.fromEntries([9, 10, 11, 12, 13].map(ecmaVersion => {
```

Comment:
We're targeting engines that don't have `Object.fromEntries`, so this'd have to be formulated differently.

Reply:
Rollup will [transpile and polyfill to ES5](https://github.com/ninofiliu/acorn/blob/dcc251b027b762f7f345a16fc71927f3c4a5fb8a/acorn/rollup.config.js#L23) anyway but in order to get the source as close as possible to the transpilation, I did a small rewrite that produces no type loss and no readability loss https://github.com/acornjs/acorn/pull/1109/commits/dcc251b027b762f7f345a16fc71927f3c4a5fb8a